### PR TITLE
Fixed `dagre-d3` and extended `dagre`

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**
- `dagre.Graph` has a method `setExtent`. There was the (optional) option field missing.
- `dagre-d3` had no exported module so there were problems with SystemJS and Atom's TypeScript package.

### CODE REVIEW

File: dagre-d3/dagre-d3-tests.ts [hunk 1]
```diff
@@ -1,19 +1,18 @@
 /// <reference path="dagre-d3.d.ts"/>
 namespace DagreD3Tests {
-    var gDagre = new dagreD3.graphlib.Graph();
-    var graph = gDagre.graph();
+    let gDagre = new dagreD3.graphlib.Graph();
```
Comment (markwongsk) [inline dagre-d3/dagre-d3-tests.ts]: if you're changing these, prefer const wherever possible.

**Comment by ChristianBeilschmidt** [comment]
@markwongsk: Thank you, you are completely right! I changed this accordingly.

**Comment by markwongsk** [comment]
Looks good to me for dagre-d3.

**Comment by markwongsk** [comment]
:+1:

**Comment by qinfchen** [comment]
looks good to me. there is a similar PR for dagre https://github.com/DefinitelyTyped/DefinitelyTyped/pull/9180.

**Comment by qinfchen** [comment]
👍 , @cpettitt

**Pull request merged**